app/controllers/audits/allocations_controller.rb

```diff
@@ -37,13 +38,20 @@ def destroy
 
   private
 
+    PAGE_SIZE = 100
+
     def user_uid
       params.fetch(:allocate_to)
     end
 
     def set_batch_value
+      page = params[:page].to_i
+      offset = page * PAGE_SIZE - PAGE_SIZE if page > 1
+
       params[:content_ids] = FindContent
-                               .paged(filter(per_page: batch_size))
+                               .paged(filter)
+                               .limit(batch_size)
```

Comment:
Again, I've partially addressed that with 5a8cfdfb6b69cb2e752bcdd0ff8a9451bc079085.

`FindContent` returns a scope, we can then apply our offset and limit to fetch the content item batch we need to allocate. I think this could be further extracted into a use case outside of the controller but again I wanted to wait until we've thought more about the concern of filtering with https://trello.com/c/seNXaaRp